widgetssdk/src/main/java/com/glia/widgets/chat/ChatView.java

```diff
@@ -810,11 +815,7 @@ public void beforeTextChanged(CharSequence charSequence, int i, int i1, int i2)
 
             @Override
             public void onTextChanged(CharSequence charSequence, int i, int i1, int i2) {
-                if (charSequence.toString().trim().length() > 0) {
-                    sendButton.setVisibility(VISIBLE);
-                } else {
-                    sendButton.setVisibility(GONE);
-                }
+
```

Comment:
extra line?

Reply:
linter is happy tho x) remains empty as the beforeTextChanged as i moved out the functionality